onboard/src/pages/WebPortal.jsx

```diff
@@ -88,7 +88,8 @@ function WebPortal() {
           child: obj.child,
           visited: true,
         };
-      } else if (obj.index === currentPageIndex + 1) {
+      } 
+      else if (obj.index === currentPageIndex + 1) {
```

Comment:
Does this need to be on a new line? If this is an auto format issue, ignore, otherwise position elses on same line: `} else if () {`